examples/standalone_single_project/main.tf

```diff
@@ -0,0 +1,263 @@
+
+}
+
+# Secure-CI
+module "ci_pipeline" {
+  source                    = "../../modules/secure-ci"
+  project_id                = local.project_id
+  app_source_repo           = "${var.app_name}-source"
+  cloudbuild_cd_repo        = "${var.app_name}-cloudbuild-cd-config"
+  gar_repo_name_suffix      = "${var.app_name}-image-repo"
+  cache_bucket_name         = "${var.app_name}-cloudbuild"
+  primary_location          = var.region
+  attestor_names_prefix     = ["build", "security", "quality"]
+  app_build_trigger_yaml    = "cloudbuild-ci.yaml"
+  runner_build_folder       = "${path.module}/cloud-build-builder"
+  build_image_config_yaml   = "cloudbuild-skaffold-build-image.yaml"
+  trigger_branch_name       = ".*"
+  cloudbuild_private_pool   = module.cloudbuild_private_pool.workerpool_id
+  clouddeploy_pipeline_name = local.clouddeploy_pipeline_name
+  skip_provisioners         = true
+}
+
+# Secure-CD
+module "cd_pipeline" {
+  source           = "../../modules/secure-cd"
+  project_id       = local.project_id
+  primary_location = var.region
+
+  gar_repo_name             = module.ci_pipeline.app_artifact_repo
+  cloudbuild_cd_repo        = "${var.app_name}-cloudbuild-cd-config"
+  deploy_branch_clusters    = local.deploy_branch_clusters
+  app_deploy_trigger_yaml   = "cloudbuild-cd.yaml"
+  cache_bucket_name         = module.ci_pipeline.cache_bucket_name
+  cloudbuild_private_pool   = module.cloudbuild_private_pool.workerpool_id
+  clouddeploy_pipeline_name = local.clouddeploy_pipeline_name
+  depends_on = [
+    module.ci_pipeline
+  ]
+}
+
+# Cloud Build Private Pool
+module "cloudbuild_private_pool" {
+  source = "../../modules/cloudbuild-private-pool"
+
+  project_id                = local.project_id
+  network_project_id        = local.project_id
+  location                  = var.region
+  create_cloudbuild_network = true
+  private_pool_vpc_name     = "cloudbuild-worker-vpc"
+  worker_pool_name          = "cloudbuild-workerpool"
+  machine_type              = "e2-highcpu-32"
+
+  worker_address    = "10.39.0.0"
+  worker_range_name = "cloudbuild-worker-range"
+}
+
+# Cloud Build Workerpool <-> GKE HA VPNs
+module "gke_cloudbuild_vpn" {
+  for_each = local.gke_net_vpn
+
+  source = "../../modules/workerpool-gke-ha-vpn"
+
+  project_id = local.project_id
+  location   = var.region
+
+  gke_project             = each.value.project_id
+  gke_network             = each.value.network
+  gke_location            = each.value.location
+  gke_control_plane_cidrs = each.value.control_plane_cidrs
+
+  workerpool_network = module.cloudbuild_private_pool.workerpool_network
+  workerpool_range   = module.cloudbuild_private_pool.workerpool_range
+  gateway_1_asn      = each.value.gateway_1_asn
+  gateway_2_asn      = each.value.gateway_2_asn
+  bgp_range_1        = each.value.bgp_range_1
+  bgp_range_2        = each.value.bgp_range_2
+}
+
+###### Private Clusters ######
+# Private Cluster VPCs
+module "vpc" {
+  for_each = toset(local.envs)
```

Comment:
Do we need 1 VPC per env? Usual pattern I have seen is 1 vpc and subnet based separation.